torchao/prototype/float8nocompile/.gitignore

```diff
@@ -1 +1,3 @@
 kernels/autogen/
+test/activation_checkpoint_test.py
```

Comment:
is this expected?

Reply:
Yes, I have a couple WIP test files for AC and FSDP I didn't want to accidentally commit them.